classes/class.tapestry.php

```diff
@@ -134,7 +134,9 @@ public function getNodeIds()
     public function addNode($node)
     {
         $tapestryNode = new TapestryNode($this->postId);
+        // $tapestryNode->update_original_author($node);
```

Comment:
I think it would be cleaner to remove code that is commented out, unless it's commented out as part of a WIP feature (in which case, I think there should be a comment stating so).